doc/release-notes/rl-2505.section.md

```diff
@@ -523,6 +523,9 @@
 
 - In `dovecot` package removed hard coding path to module directory.
 
+- `python3Packages.chromadb` has been bumped to > 1.0.0. `python3Packages.chromadb_0` contains the latest
```

Comment:
should this go to 25.11 release notes?

Reply:
It's been sitting on the old version for a long time, so I thought so. Happy to remove this if you disagree.

Reply:
My intention was to backport, if that's ok.

Reply:
I'm going to drop that plan and just keep this in unstable for now.

Reply:
Removed doc update.

Reply:
> 25.05 has already been released, modifying release notes after release seems strange

It is standard practice to modify the release notes when backporting changes that'd need release notes.

Always worth considering whether a change that needs release notes _should_ be backported, though.